File: Prácticas/Práctica11- ProoOfConceptJSONAndroid/ProoOfConceptJSON/src/org/json/test/Estudiante.java
package org.json.test;

/**
*
* @author msicu
*/
public class Estudiante
{
   public String id;
   public String nombre;
   public String apellidopaterno;
   public String apellidomaterno;
   public String matricula;
   public String direccion;
   
   public String getIdEstudinte(){
	   return id;
   }
   public String getNombreCompleto(){
	   return String.format("%s %s %s", nombre, apellidopaterno, apellidomaterno);
   }
   
   public String toString(){
	   return String.format("%s %s %s", nombre, apellidopaterno, apellidomaterno);
   }
   
}

File: Prácticas/práctica9-FrmPruebaEstudiantes/FrameworkTSCI/Model/Estudiante.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FrameworkTSCI.Model
{
    public class Estudiante:Entity
    {
        public string Nombre { get; set; }
        public string ApellidoPaterno { get; set; }
        public string ApellidoMaterno { get; set; }
        public string Matricula { get; set; }

    }
}
